docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
 
 ## Introduction
 
-Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
+One of the most powerful features of Redocly CLI is its ability to identify problems found in OpenAPI definition files. This is achieved with the `lint` command, which helps your definitions conform to a consistent set of rules.
```

Comment:
```suggestion
One of the most powerful features of Redocly CLI is its ability to identify problems found in OpenAPI definitions. This is achieved with the `lint` command, which helps your definitions conform to a consistent set of rules.
```

Reply:
I think we can keep it simple; it's generally understood that those are files.

Reply:
```suggestion
One of the most powerful features of Redocly CLI is its ability to identify problems found in OpenAPI definitions. This is achieved with the `lint` command, which keeps your API design consistent.
```